task2/Task1.java

```diff
@@ -0,0 +1,22 @@
+package task2;
+
+import java.util.Random;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int[] array = new int[10];
+        Random random = new Random();
+        long sum = 0;
+        long minNum = Long.MAX_VALUE;
+
+        for (int i = 0; i < 10; i++) {
+            array[i] = random.nextInt();
+            sum += array[i];
+            minNum = (minNum > array[i]) ? array[i] : minNum;
+            System.out.println(array[i] + " ");
```

Comment:
As I undarstand correctly you should to print all numbers in one line and separete it with space.
But here I see that you print one numper per cicle.

Reply:
Done